scripts/alcor_services.sh

```diff
@@ -52,62 +86,124 @@ function stop_remove_start_alcor_containers()
 #Function to start alcor container services
 function start_alcor_containers()
 {
+    count=0
     echo "Starting Alcor services "
     docker run --name=ignite -p 10800:10800 -p 10801:10801 -p 47100:47100 -p 47500:47500 -v /tmp:/tmp -tid ignite-11 sh
+    if [[ $? -eq 0 ]]; then count=$((count + 1)); fi
```

Comment:
What is this for?

Reply:
A tracker to count the number of containers that have successfully been started. The user should know the total number of services expected to start after the end of this step (currently 16). If the count is less than expected then we know that one or more services have not started.

Reply:
Okay, cool. We definitely need a good README then :-)